tests/integration/targets/cloudtrail/README.md

```diff
@@ -0,0 +1,8 @@
+# Cloudtrail integration test
+
+This test-suite requires too much extra privileges and we cannot run the Cloudtrail interation tests in the CI.
+Instead we run the test-suite offline with a recording of the last API interactions. This is way happen when
+we call `./run.sh`.
+
+If you introduce a change that require an update of the recording, you should call the `./record.sh` script.
+This will refresh the `recording.tar.gz` tarball with the last API interactions.
```

Comment:
Given these tests rely on recordings of 7 different modules I'm a little concerned how stable the tests will be.

Reply:
I understand your concerns. I would brand this a experimental at this stage. If we think this become a source of pain in the future I will be happy to remove this. But if it works well enough, this can safe us a lot of (CI) time.